src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Signup from "./pages/Signup";
```

Comment:
It's best to keep external imports at the top and internal imports at the bottom. You've done exactly that! Just add an extra line between the two imports to give a cleaner look